### PR TITLE
Remove point_neurite

### PR DESCRIPTION
**Pull request by mgeplf**
* we want to live in a fst world

### CODE REVIEW

File: doc/source/examples.rst [hunk 1]
```diff
@@ -57,20 +57,6 @@ Here we visualize a neuronal morphology:
     >>> fig, ax = viewer.draw(nrn, mode='3d') # valid modes '2d', '3d', 'dendrogram'
     >>> fig.show()
 
-Basic feature extraction example
```
Comment (juanchopanza) [inline doc/source/examples.rst]: Why is this example removed?

File: neurom/fst/tests/test_feature_compat.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
 
 '''Test neurom._point_neurite.features and neurom.fst features compatibility'''
```
Comment (juanchopanza) [inline neurom/fst/tests/test_feature_compat.py]: This is no longer applicable, so the name and docstring should be changed to reflect whatever this test does now.

File: doc/source/examples.rst (hunk 1 above)
Comment (mgeplf) [inline doc/source/examples.rst]: It's in the spirit of getting people to use morph_stats instead of rolling their own.

**Comment by lidakanari** [comment]
I have updated versions for most of the examples (simple ones) as they have been presented in the HBP summit. If it is useful I can add them when the clean-up is completed.

**Pull request merged**